index.html 

```diff
@@ -0,0 +1,42 @@
+<! doctype html>
+<head>
+	<title> Bubble Tea site</title>
```

Comment:
Please remove space after <title> and all similar extra spaces on other lines.
Will refer to these issues as "space tag issues"